kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveKafkaStreams.java

```diff
@@ -202,11 +202,11 @@ private ResponsiveKafkaStreams(
         responsiveConfig,
         responsiveKafkaClientSupplier,
         time,
-        storeRegistry,
         metrics,
         new SharedClients(
             cassandraClient,
-            responsiveKafkaClientSupplier.getAdmin(responsiveConfig.originals())
+            responsiveKafkaClientSupplier.getAdmin(responsiveConfig.originals()),
+            storeRegistry
```

Comment:
Just moved the store registry to the SharedClients container class, since it was already being passed around via the configs and is needed/accessed from the same places that were also accessing the other client(s)